src/devices/acurite.c

```diff
@@ -662,6 +744,117 @@ static int acurite_txr_decode(r_device *decoder, bitbuffer_t *bitbuffer)
             valid += acurite_6045_decode(decoder, bitbuffer, brow);
         }
 
+        else if ((message_type == ACURITE_MSGTYPE_ATLAS_WNDSPD_TEMP_HUM || 
+                  message_type == ACURITE_MSGTYPE_ATLAS_WNDSPD_RAIN || 
+                  message_type == ACURITE_MSGTYPE_ATLAS_WNDSPD_UV_LUX ||
+                  message_type == ACURITE_MSGTYPE_ATLAS_WNDSPD_TEMP_HUM_LTNG || 
+                  message_type == ACURITE_MSGTYPE_ATLAS_WNDSPD_RAIN_LTNG || 
+                  message_type == ACURITE_MSGTYPE_ATLAS_WNDSPD_UV_LUX_LTNG)) {
+            // {80} 82 f3 65 00 88 72 22 00 9f 95  {80} 86 f3 65 00 88 72 22 00 9f 99  {80} 8a f3 65 00 88 72 22 00 9f 9d
+            // {80} 82 f3 66 00 05 e4 81 00 9f e4  {80} 86 f3 66 00 05 e4 81 00 9f e8  {80} 8a f3 66 00 05 e4 81 00 9f ec
+            // {80} 82 f3 e7 00 00 00 96 00 9f 91  {80} 86 f3 e7 00 00 00 96 00 9f 95  {80} 8a f3 e7 00 00 00 96 00 9f 99
+            // {80} 82 f3 66 00 05 60 81 00 9f 60  {80} 86 f3 66 00 05 60 81 00 9f 64  {80} 8a f3 66 00 05 60 81 00 9f 68
+            // {80} 82 f3 65 00 88 71 24 00 9f 96  {80} 86 f3 65 00 88 71 24 00 9f 9a  {80} 8a f3 65 00 88 71 24 00 9f 9e
+            // {80} 82 f3 65 00 88 71 a5 00 9f 17  {80} 86 f3 65 00 88 71 a5 00 9f 1b  {80} 8a f3 65 00 88 71 a5 00 9f 1f
+            bitrow_printf(bb, bitbuffer->bits_per_row[brow], "%s: Acurite Atlas raw msg: ", __func__);
+            sensor_id = ((bb[0] & 0x03) << 8) | bb[1];
+            channel   = acurite_getChannel(bb[0]);
+            sprintf(channel_str, "%c", channel);
+
+            // The sensor sends the same data three times, each of these have
+            // an indicator of which one of the three it is. This means the
+            // checksum and first byte will be different for each one.
+            // The bits 4,5 of byte 0 indicate which copy
+            //  xxxx 00 xx = first copy
+            //  xxxx 01 xx = second copy
+            //  xxxx 10 xx = third copy
+            sequence_num = (bb[0] & 0x0c) >> 2;
+            // Battery status is the 7th bit 0x40. 1 = normal, 0 = low
+            battery_low = (bb[2] & 0x40) == 0;
+
+            // Wind speed is 8-bits raw MPH
+            wind_speed_mph = ((bb[3] & 0x7F) << 1) | ((bb[4] & 0x40) >> 6);
+
+            /* clang-format off */
+            data = data_make(
+                    "model",            "",         DATA_STRING, "Acurite-Atlas",
+                    "id",               NULL,       DATA_INT,    sensor_id,
+                    "channel",          NULL,       DATA_STRING, &channel_str,
+                    "sequence_num",     NULL,       DATA_INT,    sequence_num,
+                    "battery_ok",       NULL,       DATA_INT,    !battery_low,
+                    "message_type",     NULL,       DATA_INT,    message_type,
+                    "wind_avg_mi_h",    "Wind Speed", DATA_FORMAT, "%.1f mi/h", DATA_DOUBLE, wind_speed_mph,
```

Comment:
"wind_avg_mi_h" doesn't fit as a label - Doesn't follow any of the conventions for units. 

Also average probably isn't necessary since the unit doesn't report peak gusts or any other values.

Reply:
It's from our current advice on common units (https://triq.org/rtl_433/DATA_FORMAT.html#sensor-data). Hideki and Acurite also have this key.

Reply:
Ah thanks, I hadn't seen that, so are those labels recognized well enough that unit conversions, customary/SI work?

Reply:
Yes, we should have that covered with https://github.com/merbanan/rtl_433/blob/master/src/r_api.c#L537